review/communication-react.api.md

```diff
@@ -0,0 +1,1828 @@
+messages: (Message<"chat"> | Message<"system"> | Message<"custom">)[];
+}, (res1: string, res2: {
+[key: string]: ChatMessageWithStatus;
+}, res3: Date, res4: boolean) => {
+userId: string;
+showMessageStatus: boolean;
+messages: (Message<"chat"> | Message<"system"> | Message<"custom">)[];
+}>;
+
+// @public (undocumented)
+export type ChatUIState = {
+    error?: Error;
+};
+
+// @public (undocumented)
+export type ClientState = CallClientState & ChatClientState;
+
+// @public (undocumented)
+export type Common<A, B> = Pick<A, CommonProperties<A, B>>;
+
+// @public (undocumented)
+export type CommonProperties<A, B> = {
+    [P in keyof A & keyof B]: A[P] extends B[P] ? P : never;
+}[keyof A & keyof B];
+
+// @public
+export type CommunicationParticipant = {
+    userId: string;
+    displayName?: string;
+};
+
+// @public
+export interface ComponentStrings {
+    cameraButton: CameraButtonStrings;
+    endCallButton: EndCallButtonStrings;
+    errorBar: ErrorBarStrings;
+    messageStatusIndicator: MessageStatusIndicatorStrings;
+    messageThread: MessageThreadStrings;
+    microphoneButton: MicrophoneButtonStrings;
+    optionsButton: OptionsButtonStrings;
+    participantItem: ParticipantItemStrings;
+    participantsButton: ParticipantsButtonStrings;
+    screenShareButton: ScreenShareButtonStrings;
+    sendBox: SendBoxStrings;
+    typingIndicator: TypingIndicatorStrings;
+}
+
+// @public
+export const ControlBar: (props: ControlBarProps) => JSX.Element;
+
+// @public
+export const ControlBarButton: (props: ControlBarButtonProps) => JSX.Element;
+
+// @public
+export interface ControlBarButtonProps extends IButtonProps {
+    labelKey?: string;
+    onRenderOffIcon?: IRenderFunction<IButtonProps>;
+    onRenderOnIcon?: IRenderFunction<IButtonProps>;
+    showLabel?: boolean;
+    strings?: ControlBarButtonStrings;
+}
+
+// @public
+export interface ControlBarButtonStrings {
+    label?: string;
+    offLabel?: string;
+    onLabel?: string;
+}
+
+// @public (undocumented)
+export type ControlBarLayoutType = 'horizontal' | 'vertical' | 'dockedTop' | 'dockedBottom' | 'dockedLeft' | 'dockedRight' | 'floatingTop' | 'floatingBottom' | 'floatingLeft' | 'floatingRight';
+
+// @public
+export interface ControlBarProps {
+    children?: React_2.ReactNode;
+    layout?: ControlBarLayoutType;
+    styles?: BaseCustomStylesProps;
+}
+
+// @public (undocumented)
+export const createAzureCommunicationCallAdapter: ({ userId, displayName, credential, locator, callClientOptions }: AzureCommunicationCallAdapterArgs) => Promise<CallAdapter>;
+
+// @public (undocumented)
+export const createAzureCommunicationChatAdapter: ({ endpointUrl, userId, displayName, credential, threadId }: AzureCommunicationChatAdapterArgs) => Promise<ChatAdapter>;
+
+// @public (undocumented)
+export const createDefaultCallingHandlers: (callClient: StatefulCallClient, callAgent: CallAgent | undefined, deviceManager: StatefulDeviceManager | undefined, call: Call | undefined) => {
```

Comment:
Should this be an interface? e.g. `DefaulCallingHandlers`

Reply:
In fact yep it should, weh ave one for chat below

Reply:
DefaultCallingHandlers already exists below, think this just needs updated to use that as the return type

Reply:
Assigned to myself